cornice/errors.py

```diff
@@ -1,10 +1,7 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
-try:
-    import simplejson as json
-except ImportError:
-    import json
+import simplejson as json
```

Comment:
Are we still supporting Python2.6? Shouldn't we remove simplejson and use json instead?